File: leetcode/POTD/2025-02-10/solution.go
func clearDigits(s string) string {
    result := []rune{}
	for _, char := range s {
		if unicode.IsDigit(char) {
			if len(result) > 0 {
				result = result[:len(result)-1]
			}
		} else {
			result = append(result, char)
		}
	}
	return string(result)
}

File: leetcode/POTD/2025-02-10/solution.ts
function clearDigits(s: string): string {
	let result: string = ""
	for (const char of s) {
		if (/[0-9]/.test(char)) {
			if (result.length > 0) {
				result = result.slice(0, -1)
			}
		} else {
			result += char
		}
	}
	return result
}
